qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -110,17 +113,17 @@ def format(self) -> str:
         """
         status = api_status_to_job_status(ApiJobStatus(self._status)).value \
             if self._status else self._get_value(self._status)
-        estimated_start_time = arrow.get(self.estimated_start_time).humanize() \
+        est_start_time = start_duration(self.estimated_start_time) \
```

Comment:
Why did you change the arrow humanize format to start_duration format?

Reply:
Arrow humanize does the same thing that `start_duration()` does. They both find out how long it will be until the job is run, calculating the difference. On top of that, I replaced it since it removes the `arrow` dependency.

One thing I realize, however. If `start_duration()` is to be used, the name might need to be changed, since it's used on line `118` to find the approximate complete time of the job. Maybe a name such as `duration_difference`?

Reply:
The output format is slightly different though. The arrow output is more humanized and matches what IQX displays. But I suppose it's not a big deal.